src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -208,39 +207,36 @@ const JobHistory = _.flow(
               {
                 size: { basis: 150, grow: 0 },
                 headerRenderer: () => h(HeaderCell, ['Status']),
+                cellRenderer: ({ rowIndex }) => {
+                  const { workflowStatuses, status } = filteredSubmissions[rowIndex]
+                  return h(Fragment, [
+                    statusCell(workflowStatuses), _.keys(collapsedStatuses(workflowStatuses)).length === 1 && status
+                  ])
+                }
+              },
+              {
+                size: { basis: 220, grow: 0 },
+                headerRenderer: () => h(HeaderCell, ['Controls']),
```

Comment:
TOL:
```suggestion
                headerRenderer: () => h(HeaderCell, ['Actions']),
```
What do you think?

Reply:
💯 Although it wouldn't make sense in this context, "Controls" could be confused for experimental controls, especially if there's no visible content in the column to help give context.